ns/executor.go

```diff
@@ -60,7 +60,7 @@ func (nsexec *Executor) prepareCommandArgs(binary string, args, envs []string) [
 		}
 	}
 	if len(envs) > 0 {
-		cmdArgs = append(cmdArgs, "env", "-i")
+		cmdArgs = append(cmdArgs, "env")
```

Comment:
Do you know why not ignoring the existing env variables?

Reply:
The purpose of `-i` is to create an empty environment. When executing setup.sh, the env `PATH` will be cleaned up, the system utilities cannot be found anymore.
Back to question, ignoring existing env variables is not required, envs, that are required for the command, are well defined here and no need to care about other unused envs.